docs/reference/search/search-your-data/paginate-search-results.asciidoc

```diff
@@ -90,18 +96,47 @@ GET /_search
 	    "keep_alive": "1m"
   },
   "sort": [ <2>
-    {"@timestamp": "asc"},
-    {"tie_breaker_id": "asc"}
+    {"@timestamp": "asc"}
   ]
 }
 ----
 // TEST[catch:missing]
 
 <1> PIT ID for the search.
-<2> Sorts hits for the search.
+<2> Sorts hits for the search with an implicit tiebreak on `_shard_doc` ascending.
 
 The search response includes an array of `sort` values for each hit. If you used
-a PIT, the response's `pit_id` parameter contains an updated PIT ID.
+a PIT, a tiebreakeris included as the last `sort` values for each hit.
```

Comment:
Typo: `a tiebreakeris included` -> `a tiebreaker is included`

Reply:
thanks